bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -535,24 +538,29 @@ public static ManagedChannel createNettyChannel(
       ImmutableMap<String, Object> loadBalancingConfig =
           ImmutableMap.<String, Object>of("loadBalancingConfig", ImmutableList.of(grpcLbPolicy));
 
-      builder.defaultServiceConfig(loadBalancingConfig);
+      builder
+          .defaultServiceConfig(loadBalancingConfig)
+          .keepAliveTime(DIRECT_PATH_KEEP_ALIVE_TIME_SECONDS, TimeUnit.SECONDS)
+          .keepAliveTimeout(DIRECT_PATH_KEEP_ALIVE_TIMEOUT_SECONDS, TimeUnit.SECONDS);
     } else {
       builder = ManagedChannelBuilder.forAddress(host, options.getPort());
 
       if (options.usePlaintextNegotiation()) {
         builder.usePlaintext();
       }
+      builder
+          .keepAliveTime(CHANNEL_KEEP_ALIVE_TIME_SECONDS, TimeUnit.SECONDS)
+          .keepAliveTimeout(CHANNEL_KEEP_ALIVE_TIMEOUT_SECONDS, TimeUnit.SECONDS);
     }
 
     if (options.getChannelConfigurator() != null) {
       builder = options.getChannelConfigurator().configureChannel(builder, host);
     }
 
     return builder
+        .disableServiceConfigLookUp()
```

Comment:
why was this added? Do we need to test something about it?

Reply:
Service configs arent currently published for bigtable and the client doesnt expect any service configs. If at some point in the future we want to start using them then we need to ensure that the service configs in the client arent overridden by things that it doesnt expect. Unfortunately trying to test config fetching requires a dns server and very hard to achieve